dashboard/test/controllers/maker_controller_test.rb

```diff
@@ -264,6 +264,36 @@ class MakerControllerTest < ActionController::TestCase
     end
   end
 
+  test "schoolchoice: Returns full discount when applicant from Alaska" do
+    sign_in @teacher
+    school_alaska = create(:school, state: 'AK')
+
+    post :schoolchoice, params: {nces: school_alaska.id}
+    application = CircuitPlaygroundDiscountApplication.find_by(user_id: @teacher)
+
+    assert application.full_discount
+  end
+
+  test "schoolchoice: Returns full discount when applicant from Hawaii" do
+    sign_in @teacher
+    school_hawaii = create(:school, state: 'HI')
+
+    post :schoolchoice, params: {nces: school_hawaii.id}
+    application = CircuitPlaygroundDiscountApplication.find_by(user_id: @teacher)
+
+    assert application.full_discount
+  end
+
+  test "schoolchoice: Returns no discount when school not in Alaska or Hawaii" do
+    sign_in @teacher
+    school_washington = create(:school, state: 'WA')
+
+    post :schoolchoice, params: {nces: school_washington.id}
+    application = CircuitPlaygroundDiscountApplication.find_by(user_id: @teacher)
+
+    assert_equal application.full_discount, false
```

Comment:
Two notes on this line.

One: In MiniTest, the order of [`assert_equal`](https://apidock.com/ruby/MiniTest/Assertions/assert_equal)'s arguments are (expected, actual).  This matters a bit in that it makes test failures easier to read.  So here, I'd put `false` first.

Two: A neat MiniTest trick. Almost all assertions have a `refute` inverse.  It tends to read nicely. So instead of `assert_equal false, application.full_discount` you can write:
```suggestion
    refute application.full_discount
```